### PR TITLE
Valkey name in README.md in the 7.2 release

### PR DESCRIPTION
**Pull request by zuiderkwast**
This is a PR directly to the 7.2 branch.

The README is in a release tar is something some people may read. Let's do a little rebranding there. It's mostly s/Redis/Valkey/ and a lot of info about internals deleted.

I don't know if there is a commit we can cherry-pick for these changes, so I just did them again directly against the 7.2 branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,80 +123,80 @@ To build with support for the processor's internal instruction clock, use:
...
 
-In order to install Redis binaries into /usr/local/bin, just use:
+In order to install Valkey binaries into /usr/local/bin, just use:
 
     % make install
 
 You can use `make PREFIX=/some/other/directory install` if you wish to use a
 different destination.
 
-_Note_: For compatibility with Redis, we create symlinks from the Redis names (`redis-server`, `redis-cli`, etc.) to the Valkey binaries installed by `make install`.
```
Comment (vitahlin) [inline README.md:199]: Maybe Redis should be used here.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,27 @@
...
-Redis is often referred to as a *data structures* server. What this means is that Redis provides access to mutable data structures via a set of commands, which are sent using a *server-client* model with TCP sockets and a simple protocol. So different processes can query and modify the same data structures in a shared way.
-
-Data structures implemented into Redis have a few special properties:
+What is Valkey?
+---------------
 
-* Redis cares to store them on disk, even if they are always served and modified into the server memory. This means that Redis is fast, but that it is also non-volatile.
-* The implementation of data structures emphasizes memory efficiency, so data structures inside Redis will likely use less memory compared to the same data structure modelled using a high-level programming language.
-* Redis offers a number of features that are natural to find in a database, like replication, tunable levels of durability, clustering, and high availability.
+Valkey is an open source fork of Redis OSS, created at the point where Redis switched to a proprietary license.
```
Comment (madolson) [inline README.md:6]:
```suggestion
Valkey is an open source fork of Redis OSS, created at the point where Redis switched to a source-available license.
```
Some people nitpick about this.

File: README.md [hunk 3]
```diff
@@ -1,34 +1,27 @@
...
+Valkey is often referred to as a *data structures* server. What this means is that Valkey provides access to mutable data structures via a set of commands, which are sent using a *server-client* model with TCP sockets and a simple protocol. So different processes can query and modify the same data structures in a shared way.
 
-If you want to know more, this is a list of selected starting points:
+Data structures implemented into Valkey have a few special properties:
 
-* Introduction to Redis data types. https://redis.io/topics/data-types-intro
-* Try Redis directly inside your browser. https://try.redis.io
-* The full list of Redis commands. https://redis.io/commands
-* There is much more inside the official Redis documentation. https://redis.io/documentation
+* Valkey cares to store them on disk, even if they are always served and modified into the server memory. This means that Valkey is fast, but that it is also non-volatile.
```
Comment (madolson) [inline README.md:12]:
```suggestion
* Valkey can be configured store copies of the data on disk, even if they are always served and modified in memory. This means that Valkey is fast, but can also durably store the data.
```

File: README.md [hunk 4]
```diff
@@ -1,34 +1,27 @@
...
-* The implementation of data structures emphasizes memory efficiency, so data structures inside Redis will likely use less memory compared to the same data structure modelled using a high-level programming language.
-* Redis offers a number of features that are natural to find in a database, like replication, tunable levels of durability, clustering, and high availability.
+Valkey is an open source fork of Redis OSS, created at the point where Redis switched to a proprietary license.
 
-Another good example is to think of Redis as a more complex version of memcached, where the operations are not just SETs and GETs, but operations that work with complex data types like Lists, Sets, ordered data structures, and so forth.
+Valkey is often referred to as a *data structures* server. What this means is that Valkey provides access to mutable data structures via a set of commands, which are sent using a *server-client* model with TCP sockets and a simple protocol. So different processes can query and modify the same data structures in a shared way.
```
Comment (madolson) [inline README.md:8]:
```suggestion
Valkey can be thought of as a distributed data structure server.
Valkey provides access to mutable data structures via a set of commands, which are sent using a *server-client* model.
This allows different servers or processes to query and modify the same data structures in a shared way.
```

File: README.md [hunk 5]
```diff
@@ -1,34 +1,27 @@
...
 
-If you want to know more, this is a list of selected starting points:
+Data structures implemented into Valkey have a few special properties:
```
Comment (madolson) [inline README.md:10]:
```suggestion
Valkey data structures have a few special properties:
```

**Review by madolson — COMMENTED** [review]
I don't think there is any differences between unstable and 7.2 that affects this README, it's just the symlinks comment and a weird contribution comment RL made (seems foreboding now). I think we should update unstable, then copy-paste what is from unstable to here.

File: README.md (hunk 2 above)
Comment (zuiderkwast) [reply README.md:6]: Then I will nit-pick that source-available doesn't imply that it's non-open-source, which is want the text to say.

File: README.md (hunk 3 above)
Comment (zuiderkwast) [reply README.md:12]: I didn't write this text. Just changed redis to valkey. (It's probably antirez wordings, which I quite like.)

**Comment by zuiderkwast** [comment]
> I don't think there is any differences between unstable and 7.2 that affects this README, it's just the symlinks comment and a weird contribution comment RL made (seems foreboding now). I think we should update unstable, then copy-paste what is from unstable to here.

I'm fine with that. We can copy it as-is from unstable now. It looks fine. Will you do it?

It doesn't matter much to me, as long as we remove Redis in most visible headings.

File: README.md (hunk 2 above)
Comment (madolson) [reply README.md:6]: Then, "switched away form an open-source license" is precise.

File: README.md (hunk 3 above)
Comment (madolson) [reply README.md:12]: I guess I wanted to delete it because I disliked the original wording, but I guess you just want to re-add it?

Comment (zuiderkwast) [reply README.md:12]: What i did is i just edited this file on the 7.2 branch. I didn't compare to what's on unstable.

**Comment by zuiderkwast** [comment]
Superseeded by fd20a9b

**Comment by madolson** [comment]
I'll make a new issue to re-update the main page.

**Pull request closed without merging**